ucm2/conf.d/macaudio/desktops/mono/HiFi.conf

```diff
@@ -0,0 +1,59 @@
+# Mono desktops
+
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Mux' Primary"
+		cset "name='Speaker Playback Mux' Secondary"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+Define.JackDevice "0"
+If.cs42l84 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC WNF Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l84/Headphones.conf"
+	}
+}
+
+If.cs42l83 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC Boost Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l83/Headphones.conf"
+	}
+}
+
+If.havespk {
+	Condition {
+		Type ControlExists
+		Control "name='Speaker Playback Volume'"
+	}
+	True {
+		SectionDevice."Speaker" {
+			Comment "Speaker"
+
+			EnableSequence [
+				cset "name='Speaker Switch' on"
+			]
+
+			EnableSequence [
+				cset "name='ASI1 Sel' LeftRightDiv2"
+			]
+
+			Value {
+				PlaybackMixerElem "Speaker"
+				PlaybackPriority 100
+				PlaybackPCM "hw:${CardId},1"
```

Comment:
unrelated to this merge request, it strikes me odd that we use for the jack sub device a define and hard-code 1 here. Would be more consistent to use `Define.SpeakerDevice "1"` and replace the 1 here with `${var:SpeakerDevice}`